src/lidartag_cluster.cpp

```diff
@@ -132,11 +132,19 @@ void LidarTag::clusterClassifier(
 
     // Add a new cluster
     if (new_cluster->flag) {
+
       //cout << "new cluster added" << endl;
       int new_cluster_id = cluster_buff.size();
       int top_ring = point.point.ring;
       int bottom_ring = point.point.ring;
 
+      /*if (new_cluster_id == params_.debug_cluster_id &&
+        point.index == params_.debug_scan_id &&
+        point.point.ring == params_.debug_ring_id)
+      {
+        int x = 0;
+      }*/
+
```

Comment:
Whenever a new error is discovered in the lidartag the flow to identify the cause is:
 - Reproduce the error
 - Freeze the sample (identify the cluster id)
 - Set the debug parameters via rqt (cluster, ring ,scan values) by watching rviz
 - Put breakpoints in parts of the code as shown in the previous example.
 
 I want  to leave them as comments until we are completely sure we are not developing this package anymore since debugging the pointcloud detection filters is quite time-consuming. :smiling_face_with_tear:

Reply:
I see, so please add TODO or some comments describing this reason?